Added attunement rooms for Crossing and Haven

adamgoerdt opened the pull request:
Removed attunement rooms but updated Hib hometown rooms

rpherbig commented:
What do you think about putting these in a new file, maybe `base-attunement.yaml`? It would probably look very similar to `base-shrines.yaml`.

adamgoerdt commented:
Good question.  I was thinking that since they were specific to towns and
base-towns already has a way to differentiate town-specific information it
would fit in there.  I feel like doing a base-attunement would end up
duplicating the hierarchy that's already present in base-town.

On Fri, Dec 2, 2016 at 1:32 PM, Robert Herbig <notifications@github.com>
wrote:

> What do you think about putting these in a new file, maybe
> base-attunement.yaml? It would probably look very similar to
> base-shrines.yaml.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264527150>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARawoeriBEuXPKj97L818_fIw6RyLdxrks5rEGQ-gaJpZM4LCzks>
> .
>

rpherbig commented:
I think you're right that it would end up duplicating the hierarchy of base-town. But I think I'd prefer that to adding 20 * number-of-towns lines to base-town (and then doing it again for the next concept that needs to utilize hometown).

adamgoerdt commented:
base-attunement works for me.  I can write it up.  Just a simple hierarchy
with Town and an array of values?

On Fri, Dec 2, 2016 at 2:25 PM, Robert Herbig <notifications@github.com>
wrote:

> I think you're right that it would end up duplicating the hierarchy of
> base-town. But I think I'd prefer that to adding 20 * number-of-towns lines
> to base-town (and then doing it again for the next concept that needs to
> utilize hometown).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264540012>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARawoaIDfGsqS7mjBraOhbtT1e4ghRdiks5rEHCXgaJpZM4LCzks>
> .
>

rpherbig commented:
@rcuhljr 👍 or 👎 on having a separate `base-attunement.yaml`?

adamgoerdt commented:
Also, I removed the attunement lines from base-town.yaml when I fixed some
roomnubers for Hib.

On Fri, Dec 2, 2016 at 5:24 PM, Robert Herbig <notifications@github.com>
wrote:

> @rcuhljr <https://github.com/rcuhljr> 👍 or 👎 on having a separate
> base-attunement.yaml?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264578932>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARawoV1U_GSNwSAUWzZOpWoK4DJ11DqZks5rEJqWgaJpZM4LCzks>
> .
>

rcuhljr commented:
attunement is small enough I'd be fine having it in towns, but it doesn't really matter.

adamgoerdt commented:
Attunement rooms re-added for Crossing and Haven

rpherbig commented:
@adamgoerdt Will a future PR modify `attunement.lic` to make use of these rooms?

adamgoerdt commented:
That's the plan.  E said it was a 10 minute job to incorporate once the
data was broken out.  I'll probably give it a shot to practice my ruby a
bit, since it doesn't seem too difficult

On Sun, Dec 4, 2016 at 9:43 PM, Robert Herbig <notifications@github.com>
wrote:

> @adamgoerdt <https://github.com/adamgoerdt> Will a future PR modify
> attunement.lic to make use of these rooms?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264755950>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARawochP5Q_NEtpvqeyDXgHm4V0MtqxPks5rE3pogaJpZM4LCzks>
> .
>

rpherbig commented:
Awesome!

On Dec 5, 2016 8:25 AM, "adamgoerdt" <notifications@github.com> wrote:

> That's the plan. E said it was a 10 minute job to incorporate once the
> data was broken out. I'll probably give it a shot to practice my ruby a
> bit, since it doesn't seem too difficult
>
> On Sun, Dec 4, 2016 at 9:43 PM, Robert Herbig <notifications@github.com>
> wrote:
>
> > @adamgoerdt <https://github.com/adamgoerdt> Will a future PR modify
> > attunement.lic to make use of these rooms?
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264755950
> >,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/ARawochP5Q_
> NEtpvqeyDXgHm4V0MtqxPks5rE3pogaJpZM4LCzks>
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/1104#issuecomment-264853193>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABIHp4tfuqEQqXcvkvWWt4aKza2Acy6Sks5rFBDegaJpZM4LCzks>
> .
>
